kitsune/products/models.py

```diff
@@ -188,3 +188,71 @@ class Platform(ModelBase):
 
     def __str__(self):
         return "%s" % self.name
+
+
+class Banner(ModelBase):
+    uploaded_image = models.ImageField(
+        upload_to=settings.BANNER_IMAGE_PATH,
+        null=True,
+        blank=True,
+        max_length=settings.MAX_FILEPATH_LENGTH,
+        help_text="Must be 72x72px.",
+    )
+    static_image = models.CharField(
+        max_length=255,
+        blank=True,
+        help_text="Path to an image in staticfiles, e.g. 'protocol/img/logos/firefox/logo.svg'. "
+        "Takes precedence over uploaded_image, leave both blank to show no image.",
+    )
+    description = models.TextField()
+    link = models.CharField(max_length=255, blank=True)
+    button = models.CharField(
+        max_length=255,
+        blank=True,
+        help_text="Text to show on button, leave blank to show no button.",
+    )
+    modifer_class = models.CharField(
+        max_length=255,
+        blank=True,
+        help_text="Additional classes to add to the banner, space seperated.",
+    )
+
+    products = models.ManyToManyField(
+        Product,
+        blank=True,
+        help_text="Products to associate the banner with. "
+        "Leave blank to associate with all products.",
+    )
+    topics = models.ManyToManyField(
+        Topic,
+        blank=True,
+        help_text="Topics to associate the banner with. Leave blank to associate with all topics.",
+    )
+
+    visible = models.BooleanField()
+    priority = models.IntegerField(
```

Comment:
I would drop this completely. In order to assign a higher priority to one banner over another you still need to locate it and it also adds more logic. Let's just drop it completely

Reply:
If two banners would be shown in the same place, how do we decide which to show?

Reply:
Let's start with something simpler and if we ever need a second banner we can revisit